desk/app/growl.hoon

```diff
@@ -70,15 +51,29 @@
       %set-host
     ?>  =(src.bowl our.bowl)
     `this(bark-host !<(ship vase))
-      ::
+  ::
+      %default
```

Comment:
NB: I might name this `init` or `boot` or `start` or something like that instead of default.